### ISSUE TITLE
Helm chart support

### ISSUE THREAD

**Issue by ublubu** [koki/kubernetes-viewer#7]
This requires some investigation. First, I don't believe Helm charts are always valid YAML. Also, Short relies on having fully-parsed Kubernetes API objects to convert. In general, there isn't a way to convert a Helm chart to a Kubernetes API object without actually filling in the template (which is what Helm does). Logically, it would follow that there isn't a way to convert a Helm chart to anything equivalent to a Kubernetes API object (e.g. Short syntax) without doing the same.

tldr: It doesn't make sense to Short a Helm chart. There may be another way to "support" them in the Chrome extension, but I don't know what that is yet. We may just have to make a note about not converting Helm charts (e.g. on the Chrome extension store page, extension options dialog).
